.gitattributes

```diff
@@ -3,4 +3,4 @@
 
 # Tell github these are vendored files.
 # Doesn't include them in the language statistics.
-/node/_tools/suites/ linguist-vendored
+node/_tools/suites/ linguist-vendored
```

Comment:
ah, `suites` are now renamed to `test` (because some test cases uses the directory name in the asssertions)